From f249f1922a7484d870680db4a2faf2129e99b267 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Tue, 19 Dec 2017 09:18:04 -0800
Subject: [PATCH] e300: Fix use of basic_datagram_socket::native()

---
 lib/usrp/e300/e300_network.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/usrp/e300/e300_network.cpp b/lib/usrp/e300/e300_network.cpp
index f3045eb2d..fb085cdfc 100644
--- a/lib/usrp/e300/e300_network.cpp
+++ b/lib/usrp/e300/e300_network.cpp
@@ -133,7 +133,7 @@ static void e300_send_tunnel(
             if (not buff) continue;
 
             //step 2 - recv from socket
-            while (not wait_for_recv_ready(recver->native(), 100) and *running){}
+            while (not wait_for_recv_ready(recver->native_handle(), 100) and *running){}
             if (not *running) break;
             const size_t num_bytes = recver->receive_from(asio::buffer(buff->cast<void *>(), buff->size()), _rx_endpoint);
             if (E300_NETWORK_DEBUG) UHD_MSG(status) << name << " got " << num_bytes << std::endl;
-- 
2.14.5

